### PR TITLE
exceptions

### PR DESCRIPTION
**Pull request by Mulesoft-Mysti**
tested with local build after last commit.

### CODE REVIEW

File: modules/ROOT/pages/browser-support.adoc [hunk 1]
```diff
@@ -14,6 +14,13 @@ Use Anypoint Platform with the latest version of any supported browser:
 
 The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-support.adoc]:
```suggestion
Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```

File: modules/ROOT/pages/browser-support.adoc [hunk 2]
```diff
@@ -14,6 +14,13 @@ Use Anypoint Platform with the latest version of any supported browser:
...
+== Exceptions to Browser Support Policy
+
+API Community Manager and Partner Manager 2.0 each modify the browser support policy:
+
+* xref:api-community-manager::data-sheets.adoc[API Community Manager]
+* xref:release-notes::partner-manager/anypoint-partner-manager-2.0-release-notes#browser-compatibility.adoc[Partner Manager 2.0]
+
 == Unsupported Browsers
 
 Internet Explorer versions 11 and earlier are not supported by MuleSoft.
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-support.adoc]: MuleSoft > Mule runtime engine? Anypoint Platform?

File: modules/ROOT/pages/browser-support.adoc [hunk 3]
```diff
@@ -14,6 +14,13 @@ Use Anypoint Platform with the latest version of any supported browser:
 
 The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
 
+== Exceptions to Browser Support Policy
+
+API Community Manager and Partner Manager 2.0 each modify the browser support policy:
+
+* xref:api-community-manager::data-sheets.adoc[API Community Manager]
+* xref:release-notes::partner-manager/anypoint-partner-manager-2.0-release-notes#browser-compatibility.adoc[Partner Manager 2.0]
```
Comment (kbostrom) [inline modules/ROOT/pages/browser-support.adoc:22]: I think this looks fine, but if you want to be really explicit, you could use something like:

The following Anypoint Platform products use modified versions of browser support:
* API Community Manager +
See xref:api-community-manager::data-sheets.adoc[API Community Manager].
* Partner Manager 2.0 +
See xref:release-notes::partner-manager/anypoint-partner-manager-2.0-release-notes#browser-compatibility.adoc[Partner Manager 2.0].

**Review by kbostrom — APPROVED** [review]
Edited

File: modules/ROOT/pages/browser-support.adoc (hunk 2 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/browser-support.adoc]: Great catch! Anypoint Platform, thank you.

Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/browser-support.adoc]: I just deleted the prepositional phrase at the end. Given the following sentence, it seemed unnecessary.

File: modules/ROOT/pages/browser-support.adoc [hunk 4]
```diff
@@ -12,11 +12,18 @@ Use Anypoint Platform with the latest version of any supported browser:
 * Mozilla Firefox
 * Safari
 
-The Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
+Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/browser-support.adoc]: Browser support for Anypoint Platform follows the Salesforce Lightning Experience policy.
Changing to break up the four-noun noun phrase.

Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/browser-support.adoc:15]:
```suggestion
Anypoint Platform browser support follows the Salesforce Lightning Experience policy.
```

**Pull request merged**